core/web/resolver/mutation.go

```diff
@@ -393,3 +394,35 @@ func (r *Resolver) DeleteBridge(ctx context.Context, args struct {
 
 	return NewDeleteBridgePayload(&bt, nil), nil
 }
+
+func (r *Resolver) CreateVRFKey(ctx context.Context) (*CreateVRFKeyPayloadResolver, error) {
+	if err := authenticateUser(ctx); err != nil {
+		return nil, err
+	}
+
+	key, err := r.App.GetKeyStore().VRF().Create()
+	if err != nil {
+		// TODO: improve
```

Comment:
When are we dealing with this TODO?

The only error I see you could handle here is `ErrLocked`. So maybe you'd like to create an error resolver when it is locked and a key cannot be created. See `core/services/keystore/vrf.go:62`

Reply:
I think it would be better to just deal with this TODO now tbh.

Reply:
Dealing with this in an upcoming PR (see https://github.com/smartcontractkit/chainlink/pull/5376#issuecomment-964322507)